components/modeToggle.tsx

```diff
@@ -24,14 +24,17 @@ export function ModeToggle() {
         </Button>
       </DropdownMenuTrigger>
       <DropdownMenuContent align="end">
-        <DropdownMenuItem onClick={() => setTheme("light")}>
-          Light
+        <DropdownMenuItem onClick={() => setTheme("light")} className="flex items-center justify-between">
+          <span>Light</span>
+          {theme === "light" && <Check className="h-4 w-4" />}
```

Comment:
instead of comparing with string literals lets create an enum `ThemeType` with 'light' and 'dark' options